vmdb/db/migrate/20150111161446_create_nodes.rb

```diff
@@ -3,7 +3,7 @@ def up
     create_table :container_nodes do |t|
       t.string    :ems_ref
       t.string    :name
-      t.timestamp  :creation_timestamp
+      t.timestamp  :creation_timestamp, :null => true
```

Comment:
> Is the default for any timestamp column to be not-null? or is it only for the timestamps method.

It *should* be any timestamp method.

I think it's a stop gap towards Rails 5 upgrading.  In Rails 5, we're going to make timestamps default to not-null (make them default to NOW), but we can't do without forcing everyone to specify the default.  This goes back to conversations we were having about versioning migrations. /cc @matthewd